File: app/src/main/java/ar/android/lflanzoni/norrisapp/Main/MainView.java
package ar.android.lflanzoni.norrisapp.Main;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import ar.android.lflanzoni.norrisapp.Joke.jokeView;
import ar.android.lflanzoni.norrisapp.ListCategory.ListCategoriesView;
import ar.android.lflanzoni.norrisapp.R;

public class MainView extends AppCompatActivity implements View.OnClickListener {
    Button buttonRandom;
    Button buttonCategory;
    ImageView ivApp;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.mainview);
        buttonRandom = (Button) findViewById(R.id.btnRandom);
        buttonRandom.setOnClickListener((View.OnClickListener) this);
        buttonCategory =(Button) findViewById(R.id.btnCategory);
        buttonCategory.setOnClickListener((View.OnClickListener) this);
        ivApp = findViewById(R.id.iVNameApp);
        ivApp.setImageResource(R.drawable.ic_norrisicono);
    }

    @Override
    public void onClick (View v){
        switch (v.getId()){
            case R.id.btnRandom:
                Intent intent = new Intent(this, jokeView.class);
                startActivity(intent);
                break;
            case R.id.btnCategory:
                Intent i = new Intent(this, ListCategoriesView.class);
                startActivity(i);
                break;
        }
    }

}

File: app/src/main/java/ar/android/lflanzoni/norrisapp/main/MainView.kt
package ar.android.lflanzoni.norrisapp.main

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import ar.android.lflanzoni.norrisapp.R
import ar.android.lflanzoni.norrisapp.databinding.MainviewBinding
import ar.android.lflanzoni.norrisapp.joke.JokeView
import ar.android.lflanzoni.norrisapp.listCategory.ListCategoryView

class MainView: AppCompatActivity() {
    private lateinit var binding: MainviewBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = MainviewBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.iVNameApp.setImageResource(R.drawable.ic_norrisicono)

        binding.btnCategory.setOnClickListener{
            val i = Intent(this, ListCategoryView::class.java)
            startActivity(i)
        }

        binding.btnRandom.setOnClickListener {
            val intent = Intent(this, JokeView::class.java)
            startActivity(intent)
        }
    }
}